Userland/Libraries/LibWeb/FileAPI/Blob.h

```diff
@@ -48,14 +48,14 @@ class Blob
 
     virtual JS::Object* create_wrapper(JS::GlobalObject&);
 
+    ByteBuffer const& byte_buffer() const { return m_byte_buffer; }
```

Comment:
`ReadonlyBytes` pls :)

Reply:
OK, I'll look into making it ReadonlyBytes :)